sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -2582,19 +2338,8 @@ private Mono<PagedResponse<SubscriptionProperties>> listSubscriptions(String top
                     return Mono.empty();
                 }
 
-                final List<SubscriptionProperties> entities = feed.getEntry().stream()
-                    .filter(e -> e.getContent() != null && e.getContent().getSubscriptionDescription() != null)
-                    .map(e -> {
-                        final String subscriptionName = getTitleValue(e.getTitle());
-                        final SubscriptionProperties description = EntityHelper.toModel(
-                            e.getContent().getSubscriptionDescription());
+                final List<SubscriptionProperties> entities = getSubscriptionPropertiesList(topicName, feed);
```

Comment:
nit: for all the collection methods, do we need to specify that it is a list? (ie. Calling the method `getSubscriptions`)